test/storage/redis_storage_test.rb

```diff
@@ -71,7 +71,7 @@ def test_started_at
     assert_equal a, @experiment.started_at
   end
 
-  def test_cleanup
+  def test_cleanup_with_scope_argument
```

Comment:
We have mocha, so we should be able to assert the deprecation is logged.

Reply:
👍 Done